Resources/Prototypes/_LostParadise/Datasets/dreamstrings.yml

```
@@ -0,0 +1,285 @@
+﻿- type: dataset
+  id: dreamstrings
+  values:
+    #ss13
```

yamllint

[warning] missing starting space in comment

(comments)